### ISSUE TITLE
[Bug Report] [Component] [image] 版本升级到2.2.4, 导致image组件placeholder与img同时出现, 2.2.2版本正常

### ISSUE THREAD

**Issue by zhang-mol** [element-plus/element-plus#8174]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.31`
- Element Plus Version: `2.2.4`
- Browser / OS: `chrome`
- Build Tool: `Vite`

### Related Component

- `el-image`

### Reproduction Link

[Docs CN](https://element-plus.gitee.io/zh-CN/component/image.html#%E5%8A%A0%E8%BD%BD%E5%A4%B1%E8%B4%A5)

### Steps to reproduce

```vue
<el-image :src="item.src">
   <template #placeholder>
      <div class="image-slot"><span class="dot"></span></div>
   </template>
   <template #error>
      <div class="image-slot">
         <el-icon><Picture /></el-icon>
      </div>
   </template>
</el-image>
```

### What is Expected?

先出现占位符, 然后出现图片或者error

### What is actually happening?

占位符与图片(是空的img标签)同时出现, 加载完后出现图片或者error

### Additional comments

_(empty)_

![image](https://user-images.githubusercontent.com/26498888/172546176-2242aef8-2bf1-485e-9841-ca7f544ddcba.png)
![image](https://user-images.githubusercontent.com/26498888/172546639-6062e8bf-6802-48a4-9cdf-f012a1b08591.png)

**Comment by zhang-mol** [comment]
当imag的src为null时会一直这样, src有值时loading与img容器同时出现, scr加载完后才会替换掉placeholder

**Comment by zhang-mol** [comment]
例子就是官网的例子

**Comment by tolking** [comment]
Fixed in #8109 

Currently you can add styles to solve this problem

```css
.image-slot,
.image-slot {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}
```
